Source/DataSources/PolylineGeometryUpdater.js

```diff
@@ -513,8 +575,11 @@ define([
 
         this._line = line;
         this._primitives = primitives;
+        this._groundPrimitives = groundPrimitives;
+        this._groundPolylinePrimitive = undefined;
```

Comment:
Since this could be a line in a `PolylineCollection` of a `GroundPolylinePrimitive`, delay the `PolylineCollection` creation and the line added until you know which.

Reply:
This could potentially create and destroy primitives a lot if `clampToGround` is dynamic. What do you think @mramato?